otx/algorithms/classification/adapters/dor/configs/ote_custom_classification/__init__.py

```diff
@@ -0,0 +1,15 @@
+"""Model template configurations."""
```

Comment:
Do these model configs need to be in adapters? I think there will be some unused parts.

Reply:
Yes, just for dor nncf.